genestack_client/scripts/genestack_application_manager.py

```diff
@@ -622,19 +632,23 @@ def show_info(files, vendor_only, with_filename, no_filename):
 
 
 REMOVE_PROMPT = '''You are going to remove following system stable applications with version "%s":
- %s
-Do you want to continue'''
+ %s'''
 
 
 def prompt_removing_stable_version(application, apps_ids, version):
     check_tty()
-    apps = get_system_stable_apps_version(application, apps_ids, version)
+    if apps_ids:
+        apps = get_system_stable_apps_version(application, apps_ids, version)
+    else:
+        apps = application.invoke('getSystemStableIdsByVersion', version)
+
     if not apps:
         return True
 
     message = REMOVE_PROMPT % (version, '\n '.join(apps))
     try:
-        return ask_confirmation(message)
+        print message
```

Comment:
Its because `raw_input` built-in function brokes a list of applications ids. This decision was approved by AS.